superset-frontend/src/explore/components/ControlPanelsContainer.jsx

```diff
@@ -66,15 +66,14 @@ class ControlPanelsContainer extends React.Component {
   renderControl({ name, config }) {
     const { actions, controls, exploreState, form_data: formData } = this.props;
     const { visibility } = config;
+
     // If the control item is not an object, we have to look up the control data from
     // the centralized controls file.
     // When it is an object we read control data straight from `config` instead
     const controlData = {
-      ...controls[name],
       ...config,
+      ...controls[name],
       name,
-      // apply current value in formData
-      value: formData[name],
```

Comment:
This is not needed since `controls[name].value` is always in sync with `formData[name]` per https://github.com/apache/incubator-superset/blob/700429f431d20b624e886cf554b2b493793448be/superset-frontend/src/explore/reducers/exploreReducer.js#L100